### ISSUE TITLE
keditor: component has no preview

### ISSUE THREAD

**Issue by Uafinder** [Kademi/kademi-dev#6587]
![image](https://user-images.githubusercontent.com/1945230/51477920-88427c00-1d92-11e9-8a12-1cfe8719b009.png)
http://vladtest56c.admin.kademi-ci.co/pages/vladtest56cweb/version1/?contenteditor=page10.html

**Comment by bradmac** [comment]
Yep, the snippet should have lorem ipsum text like the normal html component

But not high priority, moving back to p2
